src/NovelRT/Timing/StepTimer.cpp

```diff
@@ -77,7 +77,8 @@ namespace NovelRT::Timing {
           subscriber(secondsDelta);
         }
       }
-    } else {
+    }
+    else {
```

Comment:
@RubyNova, most K&R variants want the `else` on the same line as the closing brace, yeah?

Reply:
I *believe* so, however our clang format isn't set up yet so we just kind of have both everywhere at the moment.

We should uh...probably fix that....lmao.

Reply:
Well decide which we want so I know if this comment is valid :P

Reply:
I think I would prefer it to be on the same line, just because doing K&R with it on a new line just looks strange.